www/c-icap/src/opnsense/mvc/app/models/OPNsense/CICAP/General.xml

```diff
@@ -0,0 +1,59 @@
+<model>
+    <mount>//OPNsense/cicap/general</mount>
+    <description>c-icap configuration</description>
+    <version>1.0.0</version>
+    <items>
+        <enabled type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+        </enabled>
+        <timeout type="IntegerField">
+            <default>300</default>
+            <Required>Y</Required>
+        </timeout>
+        <maxkeepaliverequests type="IntegerField">
+            <default>100</default>
+            <Required>Y</Required>
+        </maxkeepaliverequests>
+        <keepalivetimeout type="IntegerField">
+            <default>600</default>
+            <Required>Y</Required>
+        </keepalivetimeout>
+        <startservers type="IntegerField">
+            <default>3</default>
+            <Required>Y</Required>
+        </startservers>
+        <maxservers type="IntegerField">
+            <default>10</default>
+            <Required>Y</Required>
+        </maxservers>
+        <minsparethreads type="IntegerField">
+            <default>10</default>
+            <Required>Y</Required>
+        </minsparethreads>
+        <maxsparethreads type="IntegerField">
+            <default>20</default>
+	    <Required>Y</Required>
+	</maxsparethreads>
+        <threadsperchild type="IntegerField">
+            <default>10</default>
+	    <Required>Y</Required>
+	</threadsperchild>
+        <maxrequestsperchild type="IntegerField">
+            <default>0</default>
+	    <Required>Y</Required>
+	</maxrequestsperchild>
+        <listenaddress type="TextField">
+            <default>127.0.0.1</default>
+            <Required>Y</Required>
```

Comment:
I have tried it now - only a single address is supported.
So I would prefer a default of `::1` (localhost).
IPv4 should not be the default for anything since 20 years of IPv6 and there should not be any issues with IPv4 only networks as it will probably not leave the machine. Maybe this is just my preference to avoid IPv4 whenever it is possible.

Anyway you should use this field: https://github.com/opnsense/core/blob/2dcb24c5f855059a547a93fd229300679b9f944b/src/opnsense/mvc/app/models/OPNsense/Base/FieldTypes/NetworkField.php


@fichtner Is it even possible to disable IPv6? I think it's not.